pom.xml

```diff
@@ -56,12 +56,19 @@
       <artifactId>bootstrap</artifactId>
       <version>${bootstrap.version}</version>
       <scope>provided</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.webjars</groupId>
+          <artifactId>popper.js</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
 
     <!-- WebJars Dependencies - bundled as API plugins -->
     <dependency>
       <groupId>io.jenkins.plugins</groupId>
-      <artifactId>popper-api</artifactId>
+      <artifactId>popper2-api</artifactId>
+      <version>2.11.5-1</version>
```

Comment:
If I remember correctly, Bootstrap 4 does not work with Popper 2, an explicit dependency to Popper 1.x is required.

Reply:
> An explicit dependency to Popper 1.x is required.

Are you sure? Then why does the POM declare a dependency on Popper 2.5.4:

```
[INFO] +- org.webjars:bootstrap:jar:4.6.0:compile
[INFO] |  \- org.webjars:popper.js:jar:2.5.4:compile
```

Reply:
I am proceeding with the assumption that the above dependency declaration is just wrong.